libloki/storage.js

```diff
@@ -113,6 +113,33 @@
     }
   }
 
+  // fetches device mappings from server.
+  // if the device is a secondary device,
+  // fetch the device mappings for its primary device
+  async function saveAllPairingAuthorisationsFor(pubKey) {
+    const deviceMapping = await lokiFileServerAPI.getUserDeviceMapping(pubKey);
+    let { authorisations } = deviceMapping || {};
+    if (deviceMapping) {
```

Comment:
Could we exit early if `!deviceMapping` to reduce nesting and move `let { authorisations } = deviceMapping;` further down?